README.md

```diff
@@ -104,6 +104,56 @@ for Ubuntu Jammy and Swift 5.9 this would be `swift:5.9-jammy-slim`. If you'd li
 an arbitrary Ubuntu Jammy system, make sure you pass `--static-swift-stdlib` flag to `swift build`, in addition
 to the `--experimental-swift-sdk` option.
 
+## Building an SDK from a container image
+
+You can base your SDK on a container image, such as one of the
+[official Swift images](https://hub.docker.com/_/swift).   By
+default, the command below will build an SDK based on the Ubuntu
+Jammy image:
+```
+swift run swift-sdk-generator --with-docker
+```
+To build a RHEL images, use the `--linux-distribution-name` option.
+The following command will build a `ubi9`-based image: 
+``` 
+swift run swift-sdk-generator --with-docker --linux-distribution-name rhel
+```
+
+You can also specify the base container image by name:
+
+```
+swift run swift-sdk-generator --with-docker --from swift:5.9-jammy
+```
+
+```
+swift run swift-sdk-generator --with-docker --linux-distribution-name rhel --from swift:5.9-rhel-ubi9
```

Comment:
We still need to specify `--linux-distribution-name` because `copyTargetSwiftFromDocker` behaves differently for Ubuntu and RHEL images.   I think we could make it more generic and remove this requirement.

Reply:
WDYT of renaming `--from` to `--base-docker-image`?

Reply:
SGTM